Use GitHub Issues to submit Use Cases (#283)
w3c/wot-usecases: USE-CASES/README.md

## How to submit use cases

Please submit your **use case** via [Github Issues](https://github.com/w3c/wot-usecases/issues/new/choose).

***Note:*** Please refer to the [notes below](#notes-on-the-items-within-the-use-case-template) when you describe your use case.

## Review by the Use Cases TF

> 1. The proposed use case will be reviewed by the Use Cases TF.
> 1. If the use case is appoved, the Use Cases TF will include the use case into the [WoT Use Cases and Requirements Note](https://w3c.github.io/wot-usecases/).
> 1. If the use case has any problems, e.g., need further clarification about the items, the Use Cases TF will give a comment on the GitHub Issue of the use case.

Reviewer: ```suggestion
1. The proposed use case will be reviewed by the Use Cases TF.
2. If the use case is approved, the Use Cases TF will include the use case into the [WoT Use Cases and Requirements Note](https://w3c.github.io/wot-usecases/).
3. If the use case has any problems, e.g., need further clarification about the items, the Use Cases TF will give a comment on the GitHub Issue of the use case.
```

Reviewer 2: Includes @egekorkan's changes. Note that the list numbering will auto-adjust via `.md` processing.
```suggestion
1. The proposed use case will be reviewed by the Use Cases TF.
2. If the use case is approved, the Use Cases TF will add the use case to the [WoT Use Cases and Requirements Note](https://w3c.github.io/wot-usecases/).
3. If the use case has any problems, e.g., it needs further clarification about the some element(s), the Use Cases TF will request those clarification(s) in a comment on the GitHub Issue of the use case.
```

> ## Notes on the items within the Use Case Template

Reviewer: Do we really need this part? These are now in the use case template

### Submitter

Put your name here
    
### Motivation

> Provide a description of the problem that is solved by the use case and a reason why this use case is important for the users

Reviewer 2: ```suggestion
Provide a description of the problem that is solved by the use case. Include some reason(s) why this use case is important for the users.
```

    
### Description

Provide a description from the users perspective

### Expectations for stakeholders and environments

#### Target Users

List all stakeholders that are involved in the use case from the following list:
- device owners
- device user
- cloud provider
- service provider
- device manufacturer
- gateway manufacturer
- network operator (potentially transparent for WoT use cases)
- identity provider
- directory service operator

#### Expected Devices

> List the target devices, e.g. as a sensor, solar panel, air conditioner

Reviewer 2: ```suggestion
List the target device(s), e.g., a sensor, a solar panel, an air conditioner
```

#### Expected Data

> List the type of expected data, e.g. weather and climate data, medical conditions, machine sensors, vehicle data

Reviewer 2: ```suggestion
List the type(s) of expected data, e.g., weather and climate data, medical conditions, machine sensors, vehicle data
```

### Relationships with existing resources and standards

#### Dependencies on WoT- Affected WoT deliverables and/or work items

List the affected WoT deliverables that have to be changed to enable this use case

> #### Existing standards by other WGs within W3C

Reviewer 2: ```suggestion
#### Existing standards from other WGs within W3C
```

Provide links to relevant standards that are relevant for this use case

#### Existing standards outside W3C

Provide links to relevant standards that are relevant for this use case

> ### Optional clarificaitons on user's needs (optional but to be described as specific as possible so that requirements can be extracte precisely later)

Reviewer 2: ```suggestion
### Clarifications of user's needs (optional, but to be described as specifically as possible, so that precise requirements can be extracted later)
```

#### Security Considerations

* Describe any issues related to security; if there are none, say "none" and justify
* As a guide, see also the security part of [security and privacy review](https://github.com/w3c/wot-architecture/blob/main/publication/ver11/security_and_privacy.md)

#### Privacy Considerations:

* Describe any issues related to privacy; if there are none, say "none" and justify
* As a guide, see also the privacy part of [security and privacy review](https://github.com/w3c/wot-architecture/blob/main/publication/ver11/security_and_privacy.md)

#### Accessibility Considerations:

* Describe any issues related to accessibility; if there are none, say "none" and justify
* As a guide, see also [accessibility review](https://github.com/w3c/wot-architecture/blob/main/publication/ver11/accessibility.md)

> #### Internationalisation (i18n) Considerations:
> 
> * Describe any issues related to internationalization; if there are none, say "none" and justify
> * As a guide, see also [internationalization review](https://github.com/w3c/wot-architecture/blob/main/publication/ver11/internationalization.md)

Reviewer 2: Making section header match section content
```suggestion
#### Internationalization (i18n) Considerations:

* Describe any issues related to internationalization; if there are none, say "none" and justify
* As a guide, see also [internationalization review](https://github.com/w3c/wot-architecture/blob/main/publication/ver11/internationalization.md)
```

### Gaps between the user's need and what's possible today

> Describe any gaps that are not addressed in the current WoT standards and building blocks

Reviewer 2: ```suggestion
Describe any gaps that are not addressed by the current WoT standards and building blocks
```

## Terminology

We use common terminology across the use cases to make sure 
everybody has the same understanding and the descriptions match

> when we combine the use case in a single document.

Reviewer 2: ```suggestion
when we combine the use cases into a single document.
```

> Please avoid domain specific terminology, e.g. don't use MNO, telco. Better use network operator instead.

Reviewer 2: ```suggestion
Please avoid domain specific terminology, e.g., don't use "MNO", "telco". Better to use "network operator" instead.
```
